schemas/CreatePaymentReversalInput.json

```
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "effective_at": {
+      "type": "string",
+      "description": "The `Date-Time` that the payment reversal should be recorded to the account. Although the payment reversal is being added to Canopy today, it can actually be effective as of 10 days ago. Must be after the original payment being reversed.",
+      "format": "date-time",
+      "example": "2020-07-20T09:11:28+00:00"
+    },
+    "external_fields": {
+      "type": "array",
+      "nullable": true,
+      "maxItems": 10,
+      "description": "An array of static references to fields in a third party system.",
```

```suggestion
      "description": "External fields can be used to associate this record to entities in external systems.",
```

I've made this field description generic across all inputs. Does this sound okay?